src/plugins/postgrest.ts

```diff
@@ -1,16 +1,57 @@
+import { getAnonKey, getServiceKey } from '../utils/tenant'
+
+import { FastifyRequest } from 'fastify'
 import { PostgrestClient } from '@supabase/postgrest-js'
 import fastifyPlugin from 'fastify-plugin'
-import { getPostgrestClient } from '../utils'
+import { getConfig } from '../utils/config'
 
 declare module 'fastify' {
   interface FastifyRequest {
     postgrest: PostgrestClient
+    superUserPostgrest: PostgrestClient
   }
 }
 
-export default fastifyPlugin(async (fastify) => {
+async function getPostgrestClient(request: FastifyRequest, jwt: string): Promise<PostgrestClient> {
+  const { anonKey, postgrestURL, postgrestURLSuffix, xForwardedHostRegExp } = getConfig()
+  let url = postgrestURL
+  let apiKey = anonKey
+  if (xForwardedHostRegExp) {
```

Comment:
The current test also serves as truthiness narrowing since `xForwardedHostRegExp` may be undefined. But adding `isMultitenant` does make the intent clearer, done.